kili/mutations/asset/helpers.py

```diff
@@ -104,18 +104,20 @@ def upload_content(content_array: List[str], input_type, auth, project_id):
     content_type_array = []
     for content in content_array:
         if os.path.exists(content) and check_file_mime_type(content, input_type):
-            data_array.append(open(content, "rb"))
+            with open(content, "rb") as binary_data:
+                data_array.append(binary_data)
```

Comment:
the `binary_data` file pointer is closed, so you wont be able to handle it. If you want to read it later without creating the file pointer, you can simply store the filenames and open them when you want to read them.